### PR TITLE
Landscape orientation option

### PR DESCRIPTION
**Pull request by arivepr**
With the extra query param, we can now have fat pdf's!
Addressing this Jira card
Screen Shot 2022-07-22 at 5 57 25 PM

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@arivepr can you rebase the PR please?

**Review by Hyperkid123 — COMMENTED** [review]
Can you also add this option to the actual API request that should be used? The POST one?

File: server/index.tsx [hunk 1]
```diff
@@ -51,13 +57,15 @@ app.use('^/$', async (req, res, _next) => {
...
+app.post(`${APIPrefix}/generate`, async (req: PreviewHandlerRequest, res) => {
   const rhIdentity = req.headers['x-rh-identity'] as string;
+  const orientationOption = processOrientationOption(req);
 
   if (!rhIdentity) {
     return res.status(401).send('Unauthorized access not allowed');
   }
-  const template: ServiceNames = req.body.template;
+
+  const template: ServiceNames = req.query.template as ServiceNames;
```
Comment (Hyperkid123) [inline server/index.tsx:68]: The post variant of the endpoint has to get all the data from the request body, not query params. I know it's strange and we will definitely streamline it later, but this is how it works now.

Comment (arivepr) [reply server/index.tsx:68]: Hmm is there any particular reason for that? I don't have a problem changing it back of course, just want to understand!

Comment (Hyperkid123) [reply server/index.tsx:68]: There is a legacy reason from when we took over the code. AS I said we will change it in the future (soon-ish) but it has to be like this for now

Comment (arivepr) [reply server/index.tsx:68]: Ah, got it! Thanks for clarifying my friend. I'll have it up post demo 😎

**Comment by arivepr** [comment]
@Hyperkid123 Just added a new else if to catch the difference, we can take it off once we streamline how we get the orientation between all endpoints 😬

File: browser/index.ts [hunk 2]
```diff
@@ -56,6 +56,7 @@ const getNewPdfName = () => {
 export const previewPdf = async (
   url: string,
   template: ServiceNames,
+  orientationOption: boolean,
```
Comment (Hyperkid123) [inline browser/index.ts:59]: This one should be optional `orientationOption?: boolean` or have a default value.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by Hyperkid123** [comment]
@arivepr your argument order is messed up:
```sh
14:46:24 server:
14:46:24   assets by status 22.8 MiB [cached] 1 asset
14:46:24   3788 modules
14:46:24 
14:46:24   ERROR in /pdf-gen/server/index.tsx
14:46:24   ./server/index.tsx 154:6-23
14:46:24   [tsl] ERROR in /pdf-gen/server/index.tsx(154,7)
14:46:24         TS2345: Argument of type 'boolean' is not assignable to parameter of type 'Record<string, unknown>'.
14:46:24   ts-loader-default_e3b0c44298fc1c14
```

**Pull request merged**